.ci.yaml

```diff
@@ -73,17 +84,61 @@ targets:
       target_file: mac_build_all_plugins.yaml
       channel: stable
 
+  - name: Mac_arm64 macos_platform_tests master
+    bringup: true # New task
+    recipe: plugins/plugins
+    timeout: 60
+    properties:
+      channel: master
+      add_recipes_cq: "true"
+      version_file: flutter_master.version
+      target_file: macos_platform_tests.yaml
+
+  - name: Mac_arm64 macos_platform_tests stable
+    bringup: true # New task
+    recipe: plugins/plugins
+    presubmit: false
+    timeout: 60
+    properties:
+      channel: stable
+      add_recipes_cq: "true"
+      version_file: flutter_stable.version
+      target_file: macos_platform_tests.yaml
+
   ### iOS tasks ###
-  # TODO(stuartmorgan): Swap this and ios-build_all_plugins once simulator
-  # tests are reliable on the ARM infrastructure. See discussion at
-  # https://github.com/flutter/plugins/pull/5693#issuecomment-1126011089
+  # TODO(stuartmorgan): Swap the architecture of this and ios_platform_tests_*
+  # once simulator tests are reliable on the ARM infrastructure. See discussion
+  # at https://github.com/flutter/plugins/pull/5693#issuecomment-1126011089
+  - name: Mac_arm64 ios_build_all_plugins master
```

Comment:
The second space doesn't messing up the recipe script, right? I can picture a bug related to splitting on the spaces to parse `Mac_arm64`, but I didn't look.

Reply:
This is the format we've been using for all of the packages and plugins LUCI tasks, and the script has been happy with it. Infra folks helped set up the initial naming structure.